app/controllers/csv_exports_controller.rb

```diff
@@ -71,9 +71,18 @@ def deploy_filter
     end
 
     if production = params[:production].presence
-      case production
-      when 'Yes' then filter['stages.production'] = true
-      when 'No'  then filter['stages.production'] = false
+      if production == 'Yes'
+        if DeployGroup.enabled?
+          filter['environments.production'] = true
```

Comment:
logic duplication, can dry up with:
`filter_key = (DeployGroup.enabled? ? 'environments.production' : 'stages.production')`

and then 

```
filter[filter_key] = true
```